Config.yml

```diff
@@ -0,0 +1,6 @@
+partymode_enabled: False
+partymode_enabled_date: None
```

Comment:
These variables are generated during runtime and are not meant to be configured externally.